app/uk/gov/hmrc/eoricommoncomponent/frontend/CdsErrorHandler.scala

```diff
@@ -56,7 +56,9 @@ class CdsErrorHandler @Inject() (
     statusCode match {
       case NOT_FOUND                                              => Future.successful(Results.NotFound(notFoundView()))
       case BAD_REQUEST if message == Constants.INVALID_PATH_PARAM => Future.successful(Results.NotFound(notFoundView()))
-      case _                                                      => Future.successful(Results.InternalServerError(clientErrorTemplateView(message)))
+      case FORBIDDEN if message == Constants.NO_CSRF_FOUND =>
```

Comment:
Can you add a test for this please?

Reply:
I have now added the tests for the CSRF error. Please review.